rental.rb

```diff
@@ -0,0 +1,15 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly set `attr_accessor` for book and person too as you would need to both access and modify them.

Reply:
Thank you will do as requested